apps/table.html

```diff
@@ -661,21 +756,36 @@ <h3 class="text-center h3 mb-0">Available Slides</h3>
 		const oid = e.dataset.id;
 		handleDownload(oid);
 	}
+	
+	function deleteSld(e, cancel=false) {
 
-	function deleteSld(e) {
+		const userType = getUserType();
 		const oid = e.dataset.id;
 		const oname = e.dataset.name;
+		const filename = e.dataset.filename;
+		const reqId = e.dataset.reqid;
+		console.log('reqId ' + reqId);
+
 		const store = new Store('../data/');
 		if (oid) {
-			$('#confirmDeleteContent').html(`Are you sure you want to delete the slide ${oname} with id ${oid} ?`);
+			$('#confirmDeleteContent').html(`Are you sure you want to ${reqId ? 'cancel the ': ''} ${getUserType() === 'Admin' ? '' : 'request to ' } delete the slide ${oname} with id ${oid} ?`);
 			$('#deleteModal').modal('toggle');
 			$("#confirmDelete").unbind( "click" );
 			$("#confirmDelete").click(function(){
-				store.deleteSlide(oid);
+				if (userType === "Admin" && !cancel) {
```

Comment:
@Vedant1202 @birm 
I think that delete permission should also be allowed in case of DISABLE_SEC (no security)
And since usertype is "null" in that case, I think instead of using `getUserType` here, `getUserPermissions` maybe be more preferable since it uses `wcido`

Reply:
I think that's right. Want to submit a PR?

Reply:
(you could also extend #367 to include that, since it's so similar)

Reply:
> (you could also extend #367 to include that, since it's so similar)

Okay, I'll work on it.

Reply:
Yeah, that works